src/main/kotlin/step2/calculator/Expression.kt

```diff
@@ -0,0 +1,25 @@
+package step2.calculator
+
+class Expression(
+    private val prev: Double,
+    private val operator: Operator,
+    private val next: Double
```

Comment:
가급적 축약하지 않고 좀 더 명확한 네이밍을 쓰면 어떨까요? 😃
```suggestion
    private val previousNumber: Double,
    private val operator: Operator,
    private val nextNumber: Double
```